pages/_app.tsx

```diff
@@ -44,8 +46,26 @@ function MyApp({ Component, pageProps }: AppProps) {
     checkDarkTheme()
   }, [])
 
+  const productName = getProductName()
+
+  function capitalizeRouthPath() {
```

Comment:
This is not localized. Do we have localized path name inside `router` object?

Reply:
Nevermind, since `router` object doesn't contain localized path name we'll localize `pathname`